src/client/ContentAccess.js

```diff
@@ -2175,6 +2175,12 @@ exports.EmbedUrl = async function({
   // Default options
   options.controls = options.controls === undefined ? "autoHide" : options.controls;
 
+  const controlsMap = {
```

Comment:
missing hidden controls option?

Reply:
I guess 'off' is just omitted.

should support 'off with volume toggle' option though, might need support in embed app

full player opts list is here
https://github.com/eluv-io/elv-player-js/blob/main/src/index.js